### PR TITLE
[DOC] Add example for EmberArray#isAny

### PR DESCRIPTION
**Pull request by Charizard**
#18228

### CODE REVIEW

**Review by RobbieTheWagner — APPROVED** [review]

**Comment by rwwagner90** [comment]
I'm not sure if it is okay to use potentially copyrighted names for this example or not. @jenweber @MelSumner what's our policy on this?

**Comment by jenweber** [comment]
I am not sure if there is a formal policy, but in general I haven’t seen much trademarked/copyrighted stuff in the Guides/docs. Since we have an international audience and therefore international laws, it’s probably best to change the examples to something neutral or public domain.

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 1]
```diff
@@ -1100,6 +1100,28 @@ const ArrayMixin = Mixin.create(Enumerable, {
...
 
+    ```javascript
+    class Jedi {
+      constructor(name, dark) {
+        this.name = name;
+        this.dark = dark;
+      }
+    }
+    const jediKnights = [
+      new Jedi('Luke Skywalker', false),
```
Comment (MelSumner) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]: Please use three core team member's names, to be consistent with other parts of the guides. Thank you!

**Review by MelSumner — CHANGES_REQUESTED** [review]

File: packages/@ember/-internals/runtime/lib/mixins/array.js (hunk 1 above)
Comment (locks) [reply packages/@ember/-internals/runtime/lib/mixins/array.js]: You can get them from https://emberjs.com/team/ 😁

**Review by locks — CHANGES_REQUESTED** [review]

File: packages/@ember/-internals/runtime/lib/mixins/array.js [hunk 2]
```diff
@@ -1100,6 +1100,28 @@ const ArrayMixin = Mixin.create(Enumerable, {
     argument for any item in the array. This method is often simpler/faster
     than using a callback.
 
+    ```javascript
+    class Jedi {
```
Comment (mcfiredrill) [inline packages/@ember/-internals/runtime/lib/mixins/array.js]: Are native classes preferred over plain objects in the docs?
https://github.com/emberjs/ember.js/pull/18505/files

Comment (Charizard) [reply packages/@ember/-internals/runtime/lib/mixins/array.js]: @mcfiredrill I don't think so, but I know that native classes are preferred over ES6 modules. Also, I didn't notice your PR. I will close this. Thanks!

Comment (mcfiredrill) [reply packages/@ember/-internals/runtime/lib/mixins/array.js]: Thanks, I'm ok to merge whichever one is best. :)

**Pull request closed without merging**